fab/fabfile.py

```diff
@@ -1407,5 +1407,23 @@ def _execute_with_timing(fn, *args, **kwargs):
             timing_log.write('{}: {}\n'.format(fn.__name__, duration.seconds))
 
 
+def _get_github():
+    try:
+        from .config import GITHUB_APIKEY
+    except ImportError:
+        print 'Try creating a fab/config.py with your GitHub API key to automate this step.'
+        print 'See fab/config.example.py.'
+        username = raw_input('Github username: ')
+        password = getpass('Github password: ')
+        return login(
+            username=username,
+            password=password,
+        )
+    else:
+        return login(
+            token=GITHUB_APIKEY,
+        )
```

Comment:
This config step is just local, right?  What user will this be when run on captain?